business-registry-dashboard/app/components/Table/AffiliatedEntity/Action.vue

```diff
@@ -305,8 +304,12 @@ const showAmalgamateShortForm = (item: Business): boolean => {
 
 // This is called when an affiliation invitation request already exists.
 const openNewAffiliationInvite = (business: Business) => {
```

Comment:
this will emit and open the add business modal if user clicks the 'new request' table action on an existing invitation